code/modules/surgery/bodyparts/head.dm

```diff
@@ -159,3 +160,7 @@
 
 	if(standing.len)
 		return standing
+
+/obj/item/bodypart/head/burn()
```

Comment:
what about bodypart/burn()? does the same problem not occur with other limbs with organs in them?